Project/Src/Document/MarkerStrategy/MarkerStrategy.cs

```diff
@@ -34,6 +34,12 @@ public void AddMarker(TextMarker item)
             textMarker.Add(item);
         }
 
+        public void AddMarker(IList<TextMarker> items)
```

Comment:
Should it be pluralised?

Reply:
>> Yesterday when approving, I had not noticed the missing plural "s".

and I missed that only the name was changed. 

>> Igor uses to have opinions on names, too. So let's wait.

> Should it be pluralised?

Yes, #41